src/main.cpp

```diff
@@ -167,6 +169,9 @@ int main(int argc, char* argv[])
     QApplication::setAttribute(Qt::AA_ShareOpenGLContexts);
 #endif
 
+    QAccessible::installFactory(TAccessibleConsole::consoleFactory);
```

Comment:
This is being done before the `QApplication` instance is created - is this intended/required? {Some thing need to be done like this and some must not - but I do not know which applies for this...}

Reply:
Good question. I couldn’t find anything saying whether it’s one or the other.
The only thing I found was this example, where it is done after the `QApplication` instance is created:

https://doc.qt.io/qt-5/accessible-qwidget.html#implementing-interface-factories

So I changed the `installFactory()` calls to right after the `QApplication` instantiation.